app/javascript/__tests__/pages/listings/ForSale.test.tsx

```diff
@@ -22,4 +23,16 @@ describe("For Sale", () => {
     expect(asFragment()).toMatchSnapshot()
     done()
   })
+
+  it("listings with multiple listings render the first image in the array", async (done) => {
+    axios.get.mockResolvedValue({ data: { listings: [openSaleListing] } })
+
+    const { findByAltText } = render(<ForSale assetPaths="/" />)
+
+    const image = await findByAltText(`${openSaleListing.Building_Name} Building`)
+    expect(image.getAttribute("src")).toBe(
+      "https://d31h23p2gsu1kr.cloudfront.net/images/listings/a0W0P00000GlKfBUAV-4d20089ed488fce1a695b73a8a4f090d.jpg"
```

Comment:
I would need to test this further, but on first glance: this test may fail if image caching is toggled off. I think we would use the sfmohcd domain instead of cloudfront

Reply:
Great call out! I think changing out the image urls in stubbed data probably warrants its own ticket. I did, however, change the test so that it just pulls the URL right from the stubbed data, that way we only have to change it in one place.

What do you think?

Reply:
I think that is a good solution for now!